src/js/components/Select/index.d.ts

```diff
@@ -32,10 +47,10 @@ export interface BasicSelectProps {
   margin?: MarginType;
   messages?: { multiple?: string };
   name?: string;
-  onChange?: (...args: any[]) => void;
-  onClose?: (...args: any[]) => any;
+  onChange?: (event: SelectOnChangeEvent) => void;
+  onClose?: React.MouseEventHandler<HTMLButtonElement>;
   onMore?: (...args: any[]) => any;
-  onOpen?: (...args: any[]) => any;
+  onOpen?: React.MouseEventHandler<HTMLButtonElement>;
```

Comment:
the `onClose` and `onOpen` make sense to me just need to make sure we arent missing anything in the `onChange`

Reply:
`onOpen isn't called with any event so I don't think an event handler makes sense here.

If anything, it seems like `() => void` would be more appropriate for `onOpen` and `onClose`

<img width="439" alt="Screenshot 2024-10-29 at 3 05 52 PM" src="https://github.com/user-attachments/assets/145f71d9-6860-4dd2-93d2-594e32839f9a">